pkg/cmd/monitor_command/run_monitor_command.go

```diff
@@ -66,6 +67,13 @@ func NewRunMonitorCommand(ioStreams genericclioptions.IOStreams) *cobra.Command
 	cmd.Flags().StringVar(&monitorOpt.ArtifactDir,
 		"artifact-dir", monitorOpt.ArtifactDir,
 		"The directory where monitor events will be stored.")
+	cmd.Flags().StringVar(&monitorOpt.LoadBalancerType,
```

Comment:
This command is intended to be run only once against a server.  seems like the default should be all three and this will need to be a slice.

Reply:
Do we expect the monitor to start in-cluster monitor too? I assumed that the monitor doesn't change the test platform, so only one LB type can be specified so that the disruption test would orchestrate which monitors to run

Reply:
`run-monitor` now starts in-cluster monitors too